BDArmory/FX/ExplosionFX.cs

```diff
@@ -317,7 +317,10 @@ private List<BlastHitEvent> ProcessingBlastSphere()
                         if (partHit != null)
                         {
                             if (ProjectileUtils.IsIgnoredPart(partHit)) continue; // Ignore ignored parts.
-                            if (ExplosivePart != null && partHit.name == ExplosivePart.name) continue; //don't fratricide fellow missiles/bombs in a launched salvo when the first detonates
+                            if (ExplosivePart != null)
+                            {
+                                if (partHit.name == ExplosivePart.name) continue; //don't fratricide fellow missiles/bombs in a launched salvo when the first detonates
```

Comment:
I don't get what you changed here. The logic is the same as it was before.

Reply:
That was I had added a Debug.Log line in there, which needed the curly braces, when I was tracking down why the SourceVessel wasn't getting updated properly, which later got removed, but without reverting back to the original configuration

Reply:
Ah ok, no worries then.